### PR TITLE
Pine64 article and openhabian content

### PR DESCRIPTION
**Pull request by ThomDietrich**
Signed-off-by: Thomas Dietrich <Thomas.Dietrich@tu-ilmenau.de>

### CODE REVIEW

**Comment by ThomDietrich** [comment]
This PR is not finished yet. I would still be happy about a first review 😉

File: installation/openhabian.md [hunk 1]
```diff
@@ -73,32 +73,63 @@ Learn more about the Raspberry Pi as your platform for openHAB and about the req
...
-### Pine64
+### Pine A64
 
-***... Coming Soon! ...***
+We are also providing a ready-to-use system image for the Pine A64.
+The image is based on the official [Ubuntu Base Image by longsleep](http://wiki.pine64.org/index.php/Pine_A64_Software_Release) and other than the Raspberry Pi image comes as a compressed 4GB file.
+After bootup, the latest version of openHAB 2 and the mentioned settings and tools are installed.
+All packages will be downloaded in their newest version and configured to work without further modifications.
+
+Learn more about the Pine A64 as your platform for openHAB and about the requirements over in our [Pine64 article](pine.html).
```
Comment (kaikreuzer) [inline installation/openhabian.md]: -> ... about the requirements in our...

File: installation/openhabian.md [hunk 2]
```diff
@@ -73,32 +73,63 @@ Learn more about the Raspberry Pi as your platform for openHAB and about the req
...
+
+**Setup:**
+
+* [Download the latest "openHABianPine64" SD card image file](https://github.com/ThomDietrich/openhabian/releases) (note: the file is *xz* compressed)
+* Extract the compressed archive with your favorite program
+* Write the image file to your SD card (e.g. with [Etcher](https://etcher.io)), depending on your environment you might need to extract the compressed image file first (e.g. with [WinRAR](http://www.rarlab.com/download.htm) or [7zip](http://www.7-zip.org))
+* Insert the SD card into the Pine A64, connect Ethernet and power ([See here for more details](http://wiki.pine64.org/index.php/Main_Page#Step_by_Step_Instructions))
+* Wait up to **15 minutes** for openHABian to do it's magic
```
Comment (kaikreuzer) [inline installation/openhabian.md]: its

File: installation/openhabian.md [hunk 3]
```diff
@@ -73,32 +73,63 @@ Learn more about the Raspberry Pi as your platform for openHAB and about the req
...
+* Enjoy! 🎉
+
+* The device will be available under it's IP or via the local DNS name `openhabianpine64`
```
Comment (kaikreuzer) [inline installation/openhabian.md]: its IP

File: installation/openhabian.md [hunk 4]
```diff
@@ -73,32 +73,63 @@ Learn more about the Raspberry Pi as your platform for openHAB and about the req
...
+* [Connect to the openHAB 2 portal](http://docs.openhab.org/configuration/packages.html) (available after a few more minutes): [http://openhabianpine64:8080](http://openhabianpine64:8080)
```
Comment (kaikreuzer) [inline installation/openhabian.md]: dashboard, not portal

File: installation/openhabian.md [hunk 5]
```diff
@@ -73,32 +73,63 @@ Learn more about the Raspberry Pi as your platform for openHAB and about the req
...
+* [Download the latest "openHABianPi" SD card image file](https://github.com/ThomDietrich/openhabian/releases)
 * Write the image to your SD card ([instructions](https://www.raspberrypi.org/documentation/installation/installing-images/README.md))
-* Connect Ethernet, SD card and power to your Raspberry Pi
-* Wait up to **45 minutes** (setup depends on your downlink as almost everything is downloaded live)
+* Insert the SD card into the Raspberry Pi, connect Ethernet and power
+* Wait up to **45 minutes** for openHABian to do it's magic
 * Enjoy! 🎉
 
 * The device will be available under it's IP or via the local DNS name `openhabianpi`
 * [Connect to the openHAB 2 portal](http://docs.openhab.org/configuration/packages.html) (available after a few more minutes): [http://openhabianpi:8080](http://openhabianpi:8080)
```
Comment (kaikreuzer) [inline installation/openhabian.md]: dashboard, not portal

File: installation/openhabian.md [hunk 6]
```diff
@@ -73,32 +73,63 @@ Learn more about the Raspberry Pi as your platform for openHAB and about the req
...
+* [Connect to the openHAB 2 portal](http://docs.openhab.org/configuration/packages.html) (available after a few more minutes): [http://openhabianpine64:8080](http://openhabianpine64:8080)
+* [Connect to the Samba network shares](http://docs.openhab.org/installation/linux.html#mounting-locally) with `openhab:habopen`
+* If you encounter any setup problem, [please continue here](#faq-successful)
+
+You can stop reading now.
+openHABian has installed and configured your openHAB system and you can start to use it right away.
+If you want to get in touch with the system or want to install one of the previously mentioned optional components, you can come back here later.
+
+Ready for more?
+[Connect to your Pine A64 SSH console](https://www.raspberrypi.org/documentation/remote-access/ssh/windows.md) using the username and password `pi:raspberry`.
```
Comment (kaikreuzer) [inline installation/openhabian.md]: wasn't that here ubuntu:ubuntu instead of pi:raspberry?

Comment (kaikreuzer) [inline installation/openhabian.md]: Also, the link to raspberrypi.org might not be the right one here.

File: installation/openhabian.md (hunk 3 above)
Comment (ThomDietrich) [inline installation/openhabian.md]: Wow really... The difference is totally clear to me, why am I typing it wrong all the time ^^

File: installation/openhabian.md [hunk 7]
```diff
@@ -73,32 +73,62 @@ Learn more about the Raspberry Pi as your platform for openHAB and about the req
 
 **Setup:**
 
-* [Download the latest SD card image here](https://github.com/ThomDietrich/openhabian/releases)
+* [Download the latest "openHABianPi" SD card image file](https://github.com/ThomDietrich/openhabian/releases)
```
Comment (kubawolanin) [inline installation/openhabian.md]: Shouldn't the url be `https://github.com/openhab/openhabian/releases` ?

File: installation/openhabian.md [hunk 8]
```diff
@@ -73,32 +73,62 @@ Learn more about the Raspberry Pi as your platform for openHAB and about the req
...
+We also provide a ready-to-use system image for the Pine A64.
+The image is based on the official [Ubuntu Base Image by longsleep](http://wiki.pine64.org/index.php/Pine_A64_Software_Release), which comes as a compressed 4GB file.
+After boot-up the latest version of openHAB 2 and the featured settings and tools are installed.
+All packages are downloaded in their newest version and configured to work without further modifications.
+
+Learn more about the Pine A64 as your platform for openHAB and about the requirements in our [Pine A64 article](pine.html).
+
+**Setup:**
+
+* [Download the latest "openHABianPine64" SD card image file](https://github.com/ThomDietrich/openhabian/releases) (Note: the file is *xz* compressed)
```
Comment (kubawolanin) [inline installation/openhabian.md]: Shouldn't the url be `https://github.com/openhab/openhabian/releases` ?

File: installation/openhabian.md (hunk 7 above)
Comment (ThomDietrich) [inline installation/openhabian.md]: oh, good catch! It's the old URL and I've just copy&pasted it now.

**Pull request merged**